### PR TITLE
a few small tweaks

### PR DESCRIPTION
**Pull request by ncarchedi**

### CODE REVIEW

File: NOTES.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
+# general
+
+- we need to motivate this material more
```
Comment (gvwilson) [inline NOTES.md]: Addressed in #7.

File: NOTES.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+- many instructors will see teaching as a "side gig"
+- how will becoming a better teacher help them in their day job?
+- e.g. understanding how people learn will help you give more effective presentations, make more compelling arguments to colleagues, etc.
+- what part of this will be pre-reading (I suggest the the theory so the in-person session focuses on application in datacamp setting)
+- suggest we focus on courses only (no projects) for v1
```
Comment (gvwilson) [inline NOTES.md]: Agreed - reorganized section headings in index.md to reflect this.

File: NOTES.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+- should think about relationship between /authoring and this material. Seems some of this belongs in the official docs. The more duplication, the more we need to be careful to keep things in sync
```
Comment (gvwilson) [inline NOTES.md]: I think duplication is inevitable: `authoring` is the reference manual, this is the tutorial. Move 'em into the same repo in the long run?

File: NOTES.md [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+
+# novice.html
+
+## 166: diagnose and correct misconceptions
```
Comment (gvwilson) [inline NOTES.md]: Fixed by #8.

File: NOTES.md [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+
+how does this apply in datacamp context? carefully considered feedback messages? multiple choice? keep in mind we typically write SCTs, though over time we may wish to have instructors write them
+
+## 210: final exam
```
Comment (gvwilson) [inline NOTES.md]: Fixed by #8 (remove mention of final exam).

File: NOTES.md [hunk 6]
```diff
@@ -0,0 +1,127 @@
...
+
+what does this mean in datacamp context?
+
+# memory.html
+
+## 114: learning episode
```
Comment (gvwilson) [inline NOTES.md]: Fixed by #9.

File: NOTES.md [hunk 7]
```diff
@@ -0,0 +1,127 @@
...
+
+datacamp translation?
+
+## 151: quizzes and flash cards
```
Comment (gvwilson) [inline NOTES.md]: Covered in #10

File: NOTES.md [hunk 8]
```diff
@@ -0,0 +1,127 @@
...
+
+datacamp implications?
+
+## 173: parsons problems
```
Comment (gvwilson) [inline NOTES.md]: See #11.

File: NOTES.md [hunk 9]
```diff
@@ -0,0 +1,127 @@
...
+
+we mention these but don't yet support on datacamp - is there another example that's more relevant given current exercise types?
+
+# motivation.html
+
+## 36: hard to do with auto-grading
```
Comment (gvwilson) [inline NOTES.md]: Addressed in #12.

File: NOTES.md [hunk 10]
```diff
@@ -0,0 +1,127 @@
...
+
+suggestions? (maybe you'll get to this)
+
+## 99: independent validation
+
+don't understand the example given ("sure enough...")
```
Comment (gvwilson) [inline NOTES.md]: See #13.

File: NOTES.md [hunk 11]
```diff
@@ -0,0 +1,127 @@
...
+
+## 152: mix text with graphics
+
+in the videos? exercises?
```
Comment (gvwilson) [inline NOTES.md]: See #14

File: NOTES.md [hunk 12]
```diff
@@ -0,0 +1,127 @@
...
+
+## 93: activities
```
Comment (gvwilson) [inline NOTES.md]: See #15.

File: NOTES.md [hunk 13]
```diff
@@ -0,0 +1,127 @@
...
+
+should we have them use 2-4 bullets, with each mimicking an instruction?
+
+also for the deliverable, how about 1-2 exercises per chapter (4-5 chapters)? I thought that's what we went with for the specs
```
Comment (gvwilson) [inline NOTES.md]: See #16

File: NOTES.md [hunk 14]
```diff
@@ -0,0 +1,127 @@
...
+
+## sequencing
+
+can we make the deliverable format more concrete and relevant to datacamp? i.e. chapters, lessons, concepts, etc. w/ numbers based on new content guidelines
```
Comment (gvwilson) [inline NOTES.md]: see #16

File: NOTES.md [hunk 15]
```diff
@@ -0,0 +1,127 @@
...
+
+## description
+
+I like telling instructors the 1-paragraph description is more of a sales pitch to students since it forces them to think about how to motivate the material instead of just describing/summarizing it
+
+Let's give a range for number of learning objectives. e.g. 4-6?
+
+Should also make it clear prereqs are specific datacamp courses
+
+Also, worth mentioning that this will be completed with help from CL?
```
Comment (gvwilson) [inline NOTES.md]: See #17

File: NOTES.md [hunk 16]
```diff
@@ -0,0 +1,127 @@
...
+
+# course-exercises.html
+
+suggest using "exercise" instead of "question" to get them acclimated to our vocabulary
```
Comment (gvwilson) [inline NOTES.md]: See #18

File: NOTES.md [hunk 17]
```diff
@@ -0,0 +1,127 @@
...
+
+## multiple choice question
+
+"responses" --> "choices"
+
+mention that each answer ideally represents a different (incorrect) mental model? (yes, I learned something in carpentry training :))
```
Comment (gvwilson) [inline NOTES.md]: See #18 as well.

File: NOTES.md [hunk 18]
```diff
@@ -0,0 +1,127 @@
...
+
+## write and run
+
+should we use the actual exercise types here? i.e. "normal exercises" so they begin to familiarize themselves with what's available? or will this be covered later?
+
+"starter code" --> "sample code"?
```
Comment (gvwilson) [inline NOTES.md]: See #19. (I really don't like the term "normal exercise".)

File: NOTES.md (hunk 1 above)
Comment (ncarchedi) [reply NOTES.md]: merged

File: NOTES.md (hunk 3 above)
Comment (ncarchedi) [reply NOTES.md]: yes, just make sure you're following the authoring repo for now so you're aware of any updates to the docs

File: NOTES.md (hunk 4 above)
Comment (ncarchedi) [reply NOTES.md]: merged

File: NOTES.md (hunk 7 above)
Comment (ncarchedi) [reply NOTES.md]: merged

File: NOTES.md (hunk 8 above)
Comment (ncarchedi) [reply NOTES.md]: merged

File: NOTES.md [hunk 19]
```diff
@@ -0,0 +1,127 @@
...
+
+use more data science-y examples
+
+## finding bugs and tracing execution
+
+possible to use data science examples here (or just cut)?
+
+# course-build
+
+too ambitious to cover this all up front? Maybe we should split off into a second onboarding session after specs are complete (so knowledge is fresh). Regardless, instructors will need some cursory knowledge of different exercise types, auto-grading constraints, etc. to create effective course specs. Let's discuss
```
Comment (gvwilson) [inline NOTES.md]: Agreed.

File: NOTES.md (hunk 9 above)
Comment (ncarchedi) [reply NOTES.md]: merged

File: NOTES.md (hunk 10 above)
Comment (ncarchedi) [reply NOTES.md]: merged

File: NOTES.md [hunk 20]
```diff
@@ -0,0 +1,127 @@
...
+
+## architecture
+
+"sample solution" --> "solution code" to avoid confusing "sample code" and "solution code"
```
Comment (gvwilson) [inline NOTES.md]: Merged directly into master.

File: NOTES.md (hunk 11 above)
Comment (ncarchedi) [reply NOTES.md]: merged

File: NOTES.md (hunk 15 above)
Comment (ncarchedi) [reply NOTES.md]: merged

File: NOTES.md (hunk 16 above)
Comment (ncarchedi) [reply NOTES.md]: merged

File: NOTES.md [hunk 21]
```diff
@@ -0,0 +1,142 @@
...
+
+## creating the course-build
+
+normally we do this for instructors
```
Comment (gvwilson) [inline NOTES.md]: Added directly.

File: NOTES.md (hunk 18 above)
Comment (ncarchedi) [reply NOTES.md]: merged

File: NOTES.md [hunk 22]
```diff
@@ -0,0 +1,142 @@
...
+
+## build process
+
+Maybe better to focus on workflow here - something like "the master branch should always contain a working version of the course...you will work from a separate branch and create PRs for review"
```
Comment (gvwilson) [inline NOTES.md]: See #20.

File: NOTES.md [hunk 23]
```diff
@@ -0,0 +1,142 @@
...
+
+need to update according to new authoring syntax
+
+## typical exercise
+
+same as above
+
+## note on languages
+
+some of this should probably make it into the official authoring docs at /authoring
```
Comment (gvwilson) [inline NOTES.md]: agreed

File: NOTES.md [hunk 24]
```diff
@@ -0,0 +1,142 @@
...
+
+normally we do this for instructors
+
+## build process
+
+Maybe better to focus on workflow here - something like "the master branch should always contain a working version of the course...you will work from a separate branch and create PRs for review"
+
+## chapters and exercises
+
+need to update according to new authoring syntax
```
Comment (gvwilson) [inline NOTES.md]: see #21

File: NOTES.md [hunk 25]
```diff
@@ -0,0 +1,142 @@
...
+
+## typical exercise
+
+same as above
```
Comment (gvwilson) [inline NOTES.md]: See #21

File: NOTES.md [hunk 26]
```diff
@@ -0,0 +1,142 @@
...
+
+## write and run
+
+should we use the actual exercise types here? i.e. "normal exercises" so they begin to familiarize themselves with what's available? or will this be covered later?
+
+"starter code" --> "sample code"?
+
+## minimal changes
+
+use more data science-y examples
```
Comment (gvwilson) [inline NOTES.md]: see #22

File: NOTES.md [hunk 27]
```diff
@@ -0,0 +1,142 @@
...
+
+## finding bugs and tracing execution
+
+possible to use data science examples here (or just cut)?
```
Comment (gvwilson) [inline NOTES.md]: see #22

File: NOTES.md [hunk 28]
```diff
@@ -0,0 +1,142 @@
...
+
+some of this should probably make it into the official authoring docs at /authoring
+
+## course structure, etc.
+
+TODO: nick to review when back online - hard to view in html
+
+# course editor
+
+vision here? how much do we want to get into this given the product will change? maybe better to create short screencasts for just-in-time docs?
```
Comment (gvwilson) [inline NOTES.md]: See #23.

File: NOTES.md [hunk 29]
```diff
@@ -0,0 +1,142 @@
...
+
+# course-guidelines
+
+- "First chapter is shorter because it's the funnel" --> and we want to give students a sense of accomplishment early on, to motivate them to continue
+- "1-4 instructions per exercise"; "10-480 characters (2-80 words) per instruction" --> doesn't add up?
+- "Starter Code" --> "Sample Code"
+- for sample code, does the 0-10 lines include white space? I forget what was decided here
+- in commenting example, we don't normally use periods in comments (unless separating two short sentences)
+- "1-4 hints per exercise" --> mention these often map 1-1 with instructions?
```
Comment (gvwilson) [inline NOTES.md]: See #24

File: NOTES.md (hunk 26 above)
Comment (gvwilson) [reply NOTES.md]: See #26 for a fix.

**Pull request merged**